lang/nodejs14/Portfile

```
@@ -50,14 +46,18 @@ checksums               rmd160  fb95405a2acaee5064a021e2dc8a6a6493964360 \
 
 distname                node-v${version}
 
-depends_build-append    port:pkgconfig
-
 set py_ver              3.10
 set py_ver_nodot        [string map {. {}} ${py_ver}]
 
+depends_build-append    port:pkgconfig
```

The build is failing because python310 was not found.
```suggestion
depends_build-append    port:pkgconfig \
                        port:python${py_ver_nodot}
```